### DISCUSSION TITLE
How to access audit.log?

### DISCUSSION THREAD

**Discussion by cyberwolfie1** [nextcloud/all-in-one#3715] [Questions] [answered]
This might be a very basic question, but how would I access audit.log when running Nextcloud AIO?

**Comment by szaimen** [comment, marked as answer]
Hi, the audit log is b, default stored here:
`/var/lib/docker/volumes/nextcloud_aio_nextcloud/_data/data/audit.log`

**Reply by cyberwolfie1** [reply]
Perfect, thank you :)

**Reply by sunnyd24** [reply]
@szaimen My audit.log file in location above is empty, is this because my log level by default is set to warn only? 
How can I change the log level in AIO to info or debug?

**Comment by CreativePR** [comment]
Is there a way to have this displayed in the admin logging panel in Nextcloud AIO?

**Reply by szaimen** [reply]
Currently, there isn't afaik.

**Reply by CreativePR** [reply]
I found an OCC command that let us route audit.log messages to nextcloud.log which allows most of it to be viewed in the admin panel.

Thanks

**Reply by sunnyd24** [reply]
> I found an OCC command that let us route audit.log messages to nextcloud.log which allows most of it to be viewed in the admin panel.
> 
> Thanks

@CreativePR It seems you have a solution to this, thanks! 
What was the OCC command to route audit.log messages to nextcloud.log please?

**Reply by CreativePR** [reply]
> > I found an OCC command that let us route audit.log messages to nextcloud.log which allows most of it to be viewed in the admin panel.
> > Thanks
> 
> @CreativePR It seems you have a solution to this, thanks! What was the OCC command to route audit.log messages to nextcloud.log please?

sudo docker exec --user www-data nextcloud-aio-nextcloud php occ config:app:set admin_audit logfile --value=/var/www/html/data/nextcloud.log

**Reply by derStephan** [reply]
will this persist over server reboots?

**Reply by CreativePR** [reply]
> will this persist over server reboots?

No it will not
